dpe/src/commands/destroy_context.rs

```diff
@@ -91,4 +99,144 @@ mod tests {
             Command::deserialize(&command)
         );
     }
+
+    #[test]
+    fn test_destroy_context() {
+        let mut env = DpeEnv::<TestTypes> {
+            crypto: OpensslCrypto::new(),
+            platform: DefaultPlatform,
+        };
+        let mut dpe = DpeInstance::new(&mut env, Support::default()).unwrap();
+
+        InitCtxCmd::new_use_default()
+            .execute(&mut dpe, &mut env, TEST_LOCALITIES[0])
+            .unwrap();
+
+        // Wrong locality.
+        assert_eq!(
+            Err(DpeErrorCode::InvalidLocality),
+            DestroyCtxCmd {
+                handle: ContextHandle::default(),
+                flags: DestroyCtxFlags::empty(),
+            }
+            .execute(&mut dpe, &mut env, TEST_LOCALITIES[1])
+        );
+
+        // create two dummy contexts at indices 0 and 1, with 1 being the child of 0
+        activate_dummy_context(&mut dpe, 0, Context::ROOT_INDEX, &TEST_HANDLE, &[1]);
+        activate_dummy_context(&mut dpe, 1, 0, &ContextHandle::default(), &[]);
+        // destroy context[1]
+        assert_eq!(
+            Ok(Response::DestroyCtx(ResponseHdr::new(
+                DpeErrorCode::NoError,
+            ))),
+            DestroyCtxCmd {
+                handle: ContextHandle::default(),
+                flags: DestroyCtxFlags::empty(),
+            }
+            .execute(&mut dpe, &mut env, TEST_LOCALITIES[0])
+        );
+        assert_eq!(dpe.contexts[1].state, ContextState::Inactive);
+        assert_eq!(dpe.contexts[0].children, 0);
+        // destroy context[0]
+        assert_eq!(
+            Ok(Response::DestroyCtx(ResponseHdr::new(
+                DpeErrorCode::NoError,
+            ))),
+            DestroyCtxCmd {
+                handle: TEST_HANDLE,
+                flags: DestroyCtxFlags::empty(),
+            }
+            .execute(&mut dpe, &mut env, TEST_LOCALITIES[0])
+        );
+        assert_eq!(dpe.contexts[0].state, ContextState::Inactive);
+
+        activate_dummy_context(
+            &mut dpe,
+            0,
+            Context::ROOT_INDEX,
+            &ContextHandle([0; ContextHandle::SIZE]),
```

Comment:
nit: it might be worthwhile to use `ContextHandle::default()` here to make it obvious which one is being destroyed later.

Reply:
Done.